OpenChange/RTFHandler.m

```diff
@@ -660,22 +660,17 @@ - (void) parseStyleSheet
    } while (count != 0); 
 }
 
-//
-//
-//
-- (void) parsePicture
+- (void) parseIgnoringEverything
 {
+  unsigned int count = 1;
```

Comment:
This is important, we will parse stuff until we find `}`. All rtf items are of the form: `{\whatever..... }` and when this function is called we have already parsed the first `{`